src/ledger/backend/ethereumOnChainQuerier.spec.ts

```diff
@@ -0,0 +1,155 @@
+"use strict";
+
+import { expect } from "chai";
+import { EthereumOnChainQuerier } from "./ethereumOnChainQuerier";
+import { OnChainQuerier } from "../onChainQuerier";
+
+import * as test from "#erdstall/test";
+import { Environment, setupEnv } from "test/ledger";
+import { Wallet } from "@ethersproject/wallet";
+import { IERC721Minter__factory, IERC721__factory } from ".";
+import { Tokens } from "../assets";
+import PRNG from "#erdstall/test/random";
+import { afterEach } from "mocha";
+
+const TOKEN_SIZE = 4;
```

Comment:
```suggestion
const PRIMES = [2, 3, 5, 7];
const TOKEN_SIZE = PRIMES.length;
```

Force whoever comes after you to think about the test setup and tie implicit assumption like that together. This is in context to the way you loop over the `{a,b}TransferSeed`s below.